products/compute/api.yaml

```diff
@@ -2870,6 +2870,55 @@ objects:
         imports: 'selfLink'
         description: |
           A reference to the BackendService resource.
+  - !ruby/object:Api::Resource
+    name: 'TargetVpnGateway'
+    exclude: true
+    kind: 'compute#targetVpnGateway'
+    base_url: projects/{{project}}/regions/{{region}}/targetVpnGateways
+    exports:
+      - !ruby/object:Api::Type::SelfLink
+        name: 'selfLink'
+    description: |
+      Represents a VPN gateway running in GCP. This virtual device is managed
+      by Google, but used only by you.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      api: https://cloud.google.com/compute/docs/reference/rest/v1/targetVpnGateways
+<%= indent(compile('templates/regional_async.yaml.erb'), 4) %>
+    parameters:
```

Comment:
Should we just go ahead and add all the other properties that TF doesn't have? (tunnels, forwarding rules, etc.)

Reply:
Done